supervision/detection/core.py

```diff
@@ -170,39 +170,6 @@ def from_yolov5(cls, yolov5_results) -> Detections:
             class_id=yolov5_detections_predictions[:, 5].astype(int),
         )
 
-    @classmethod
```

Comment:
Bring back that implementation. We are going to drop it with `supervision-0.15.0`, so not in the next release.

Reply:
Done!. Makes sense